anta/tests/interfaces.py

```diff
@@ -219,30 +224,17 @@ class Input(AntaTest.Input):
 
         interfaces: list[InterfaceState]
         """List of interfaces with their expected state."""
-
-        class InterfaceState(BaseModel):
-            """Model for an interface state."""
-
-            name: Interface
-            """Interface to validate."""
-            status: Literal["up", "down", "adminDown"]
-            """Expected status of the interface."""
-            line_protocol_status: Literal["up", "down", "testing", "unknown", "dormant", "notPresent", "lowerLayerDown"] | None = None
-            """Expected line protocol status of the interface."""
+        InterfaceState: ClassVar[type[InterfaceState]] = InterfaceState
```

Comment:
should there be a TODO to remove this later? (2.0.0)?

Reply:
We should track it as an issue but not in the code (trying to keep tests clean :))